File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.java
class Solution {
    public int titleToNumber(String columnTitle) {
        int x = 0;
        char[] chars = columnTitle.toCharArray();
        for (char c : chars) {
            int value = c - 'A' + 1;
            x = x * 26 + value;
        }
        return x;
    }
}

File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.py
class Solution(object):
    def titleToNumber(self, columnTitle):
        """
        :type columnTitle: str
        :rtype: int
        """
        x = 0
        for c in columnTitle:
            value = ord(c) - ord('A') + 1
            x = x * 26 + value
        return x

        